setup.cfg

```diff
@@ -0,0 +1,2 @@
+[bdist_wheel]
```

Comment:
Yeah, it's also documented on http://pythonwheels.com/.

Just curious why this changed. :)